arduino/SysConfig_Rover.h

```diff
@@ -47,3 +40,9 @@
 
 // BNO055 IMU
 #define HAS_BNO055 (0)
+
+// Receiver
+#define HAS_RECEIVER (0)
+
+//Some_module
+#define HAS_SOME_MODULE (0)
```

Comment:
Double check that the correct modules are being enabled and disabled.

Reply:
Absolutely we will.  For now, let's keep all the modules turned off.  We'll turn them on as we build out the surface and rover functionality.